tapdb/addrs_test.go

```diff
@@ -168,6 +171,11 @@ func TestAddressInsertion(t *testing.T) {
 	dbAddrs, err := addrBook.QueryAddrs(ctx, address.QueryParams{})
 	require.NoError(t, err)
 
+	// Set proof courier addresses on dbAddrs.
+	for i := range dbAddrs {
```

Comment:
So if we explicitly store a proof courier address, this shouldn't be necessary. So nowhere in this file I would expect the `ProofCourierAddr` member of an address struct being overwritten. Instead I think we need to update the `addrs` and add the address there as a new column.